vision/src/autogluon/vision/predictor/predictor.py

```diff
@@ -55,20 +59,28 @@ class ImagePredictor(object):
     # Dataset is a subclass of `pd.DataFrame`, with `image` and `label` columns.
     Dataset = ImageClassification.Dataset
 
-    def __init__(self, label='label', problem_type=None, eval_metric=None, path=None, verbosity=2):
-        warnings.warn(
+    def __init__(self, label='label', problem_type=None, eval_metric=None, path=None, backend='vision', verbosity=2):
+        self._verbosity = verbosity
+        logger.warning(
             f"AutoGluon ImagePredictor will be deprecated in v0.7. "
             f"Please use AutoGluon MultiModalPredictor instead for more functionalities and better support. "
-            f"Visit https://auto.gluon.ai/stable/tutorials/multimodal/index.html for more details!",
-            DeprecationWarning,
+            f"Visit https://auto.gluon.ai/stable/tutorials/multimodal/index.html for more details! "
         )
+
+        if backend == 'automm':
+            self._classifier = MultiModalPredictor(label=label, problem_type=problem_type, eval_metric=eval_metric,
```

Comment:
Here checkpoint_name is not specified, also we shall set `pipeline` to `OBJECT_DETECTION` (or `problem_type` if we want to remove `pipeline`)

Reply:
checkpoint_name will be provided when calling `fit`. Here is just initializing the predictor. 

For object detection, we are not using this predictor.py file. There is another file in ag.vision called detector.py (https://github.com/awslabs/autogluon/blob/master/vision/src/autogluon/vision/detector/detector.py).

Reply:
I see, but if we use automm as backend, in `MultiModalPredictor` the pipeline flag is used, and the `MMDetAutoModelForObjectDetection` (in mmdet_image.py) is initialized in `MultiModalPredictor.__init__`.

Reply:
I might have a misunderstanding. Is this only for timm and not for mmdet in 0.6?